gateway-ha/src/main/java/io/trino/gateway/ha/router/HaGatewayManager.java

```diff
@@ -110,7 +117,13 @@ public ProxyBackendConfiguration updateBackend(ProxyBackendConfiguration backend
             dao.create(backend.getName(), backend.getRoutingGroup(), backendProxyTo, backendExternalUrl, backend.isActive());
         }
         else {
+            Boolean prevStatus = dao.findFirstByName(backend.getName()).active();
             dao.update(backend.getName(), backend.getRoutingGroup(), backendProxyTo, backendExternalUrl, backend.isActive());
+            Boolean currStatus = dao.findFirstByName(backend.getName()).active();
```

Comment:
We don't need to re-fetch `currStatus`, it will be equal to `backend.isActive()`, right?